cc_plugin_ugrid/resources/adcirc_reference.cdl

```diff
@@ -0,0 +1,53 @@
+netcdf adcirc_reference {
```

Comment:
change to match the associated .nc file:

`netcdf adcirc {`

and update .cdl filename, better to have a clear map between what is responsible for .nc

is this .cdl the dump of existing adcirc file?